### ISSUE TITLE
Could not parse or process unattend file in specialize part

### ISSUE THREAD

**Issue by markgyoni** [cschneegans/unattend-generator#6]
Hey, I've just started trying out the new script options and something is returning an error in my specialize part. No custom fields, I only changed the wifi password.

xml: https://gist.github.com/markgyoni/e646d584b96c8c9ca037460b21215fe1

Thanks in advance

Edit: Turns out it's the wifi part that has the issue. I have a space in my ssid could that be the error?

**Comment by cschneegans** [comment]
The problem that causes Windows Setup to show the fatal _Windows could not parse or process…_ message is almost certainly due to the reformatting/rewrapping of the .xml file. This introduces additional whitespace (line breaks and tabs) which increase the length of some `<Path>…</Path>` elements' values to more than the allowed 256 characters.

Reformatting/rewrapping might also change the semantics of certain commands. For example,

```
<Path>cmd.exe /c "&gt;&gt;"%TEMP%\wifi.xml" echo ^&lt;name^&gt;Tbone WiFi^&lt;/name^&gt;"</Path>
```
and
```
<Path>cmd.exe /c "&gt;&gt;"%TEMP%\wifi.xml" echo ^&lt;name^&gt;Tbone
	WiFi^&lt;/name^&gt;"</Path>
```

will probably not produce the same result.

In short, avoid reformatting/rewrapping the autounattend.xml file.

**Comment by markgyoni** [comment]
Seems like the logical solution, since Prettier probably auto formatted when I edited with vscode. I'll update and close once I test it out.

**Comment by markgyoni** [comment]
Forgot to update, solution works.
